services/ext/api.go

```diff
@@ -1358,6 +1358,13 @@ func (api *PublicAPI) DeleteActivityCenterNotifications(ctx context.Context, ids
 	return err
 }
 
+// FetchNewsMessages fetches news messages from the News Feed
+// and returns a MessengerResponse containing the AC notifications
+func (api *PublicAPI) FetchNewsMessages() (*protocol.MessengerResponse, error) {
+	m := api.service.messenger
+	return m.FetchNewsMessages()
+}
+
```

Comment:
Why do we need it in public API? Aren't we automatically fetching it in the background?

Reply:
It's for Mobile. They need an API to "force" fetch, because they will use a Push notification server that will signal when there is a new News and the polling is only every 30 minutes, so they need to call the fetch function manually when the push notification comes before.